msg/tilting_mc_desired_angles.msg

```diff
@@ -0,0 +1,6 @@
+uint64 timestamp		# time since system start (microseconds)
+
+float32 roll_body		# body angle in NED frame
+float32 pitch_body		# body angle in NED flag_control_manual_enabled
```

Comment:
Why is `yaw` left to be free?

Reply:
I didn't add the yaw at the moment because it is intended to work in offboard or position flight mode. In the first one, the yaw is already sent with the normal trajectory message and I didn't want to mess up with the rest, while in the second the yaw angle is sent from the RC.
Of course, it can be changed/improved